jsonnet/rules.libsonnet

```
@@ -111,7 +111,7 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
                   cluster:master_nodes
                   * on(node) group_left() max by(node)
                   (
-                    kube_node_status_capacity_cpu_cores
+                    kube_node_status_capacity{unit="core"}
```

it might be more explicit to scope down to the "cpu" resource label too.
 
```suggestion
                    kube_node_status_capacity{resource="cpu",unit="core"}
```